src/router/index.js

```diff
@@ -15,7 +15,8 @@ const routes = [
     path: '/',
     component: () => import('layouts/GiscubeLayout.vue'),
     children: [
-      { path: '', component: HomePanel, name: 'home' },
+      { path: '', redirect: { name: 'home' } },
+      { path: 'home/', component: HomePanel, name: 'home' },
```

Comment:
why is this change needed?

Reply:
With the original version the `q-item` for home is always selected because the route always starts with `''`. By redirecting it to `/home` and then assign it to the `HomePanel`, the `q-item` is only hightlighted when you have the home component